R/train_models.R

```diff
@@ -288,7 +288,6 @@ train_models <- function(run_info,
           dplyr::select(Model_Name, Model_Recipe) %>%
           dplyr::group_split(dplyr::row_number(), .keep = FALSE),
         .combine = "rbind",
-        .errorhandling = "remove",
```

Comment:
please add this back, if a model fails we want to not raise an error. Since some models will fail based on the time series provided. But if all models fail for a time series, then we return an error at the end.